<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiFriends</class>
 <widget class="QDialog" name="uiFriends">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>309</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>367</width>
    <height>309</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>367</width>
    <height>309</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Friends list</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/logo_64.png</normaloff>:/images/logo_64.png</iconset>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>On-line</string>
       </attribute>
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>351</width>
          <height>231</height>
         </rect>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Off-line</string>
       </attribute>
       <widget class="QListWidget" name="listWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>351</width>
          <height>231</height>
         </rect>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
